components/LoginForm.vue

```diff
@@ -72,16 +67,15 @@ export default {
         this.$auth.setUser(response.data.User);
       } catch (error) {
         console.error('Failed to login:', error);
+        let errorMessage = 'Unexpected Error';
         if (!error.response) {
-          this.errorMessage = 'Server Not Found';
+          errorMessage = 'Server Not Found';
         } else if (error.response.status === 500) {
-          this.errorMessage = 'Incorrect Password';
+          errorMessage = 'Incorrect Password';
         } else if (error.response.status === 400) {
-          this.errorMessage = 'Bad Request. Try Again';
-        } else {
-          this.errorMessage = 'Unexpected Error';
+          errorMessage = 'Bad Request. Try Again';
         }
-        this.errorMessageSnackbar = true;
+        this.$snackbar(errorMessage, 'red');
```

Comment:
```suggestion
        this.$snackbar(errorMessage, 'error');
```

That way, it adapts to the user's color scheme.